pip_build.py

```diff
@@ -49,14 +50,34 @@ def ignore_files(_, filenames):
     return [f for f in filenames if "test" in f]
 
 
-def build():
+def export_version_string(version, is_nightly=False):
```

Comment:
It looks like our version string in `__init__.py` is the current release version. Maybe we should update that to `0.7.0` so it is pointing to the next major version we will release?

Reply:
Done